transforms/he/he_activity_type.py

```diff
@@ -0,0 +1,20 @@
+# uses 000 Data Ingest > 20210308 activity2activity_type.xlsx
+{
+    'sheet': 'activity2activity_type',
+    'non_unique': ['vm:hasActivity'],
+    'allow_empty_subject': True,
+    'lets': {
+        'type_iri': 'vm:HE/activitytype-{row[activity_type].as_slug}',
+        'activity_iri':  'vm:HE/{row[ActivityID].as_slug}'
+    },
+    'triples': [
+        ('{type_iri}', 'rdf:type', 'vm:HE/ActivityType'),
+        ('{type_iri}', 'vm:name', '{row[activity_type].as_text}'),
+        ('{activity_iri}', 'vm:activityType', '{type_iri}'),
+        # two way relationship requested but not in ontology, so wing it
+        ('{type_iri}', 'vm:hasActivity', '{activity_iri}'),
```

Comment:
Having this is probably just wrong - unless we're claiming to enumerate a complete set of activities. Just the link from activity to type should be sufficient.

Reply:
My thinking on this is that there's not much point in adding this data (or indeed a lot of these classes) as-is because key relationships are missing. The ontology states that we have a one-way Activity -> ActivityType relationship, and then ActivityType has a two-way relationship with Capability, which would be a cool way to link between Activity view and Capability view (or at least would be if the view switched automatically depending on what you had selected...)  -- but the Capability link is missing. So currently we can go from an Activity to an ActivityType and then you can't go from the ActivityType to anywhere else because nothing is linked to it.  It's literally just a node with a name and nothing else.  

Now, whether attempting to enumerate all Activities within a single ActivityType is a useful and sane thing to be doing, I could not tell you. But currently it is the only thing that you _can_ do with this data from a map point of view.

I'll circle back around on this after the data catch-up and delete (or not) after I find out what's going to be happening with the missing data.

Reply:
Merging this in as-is, I'll see how it looks when combined with all of the other data and then modify in a coming transforms-tweaking PR if needed.